packages/orbit-components/src/InputField/__tests__/index.test.jsx

```diff
@@ -64,7 +64,7 @@ describe("InputField", () => {
     expect(screen.getByTestId("prefix")).toBeInTheDocument();
     expect(screen.getByTestId("suffix")).toBeInTheDocument();
     fireEvent.focus(input);
-    expect(screen.getByTestId("help")).toBeInTheDocument();
+    await waitFor(() => expect(screen.getByTestId("help")).toBeInTheDocument());
```

Comment:
What do you think about using `FindBy` queries instead? It's shorter.
```suggestion
    expect(await screen.findByTestId("help")).toBeInTheDocument();
```
It's not symmetrical to waiting for disappearance, if that bothers you you can keep using `waitFor`.